Integrations/integration-rasterize.yml

```diff
@@ -56,6 +56,45 @@ script:
             demisto.results(error_message)
         sys.exit(0)
 
+    if demisto.command() == 'rasterize-image':
+        res = demisto.getFilePath(demisto.args()['EntryID'])
+        # demisto.results({'res':res})
+        # res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
+        with open(res['path'], 'r') as f:
+            data = f.read()
+        b64 = base64.b64encode(data)
+        html = '<img src="data:image/png;base64, ' + b64 + '">'
```

Comment:
Why PNG and not use file magic to get actual type? It can be JPEG or GIF as well.